src/servers/database.ts

```diff
@@ -357,7 +357,7 @@ export class DatabaseServer extends SkeletonServer {
         't.hash AS "transactionHash"',
         'e.index AS "logIndex"',
         't.to AS "address"',
-        'e.topics AS "topics"',
+        'string_to_array(concat(\'0x\',encode(e.topics[1], \'hex\'), \',\', \'0x\', encode(e.topics[2], \'hex\'), \',\', \'0x\', encode(e.topics[3], \'hex\'), \',\', \'0x\', encode(e.topics[4], \'hex\')), \',\') AS "topics"',
```

Comment:
What is the difference between previous line and this line.

Also, I see there are not tests for this repository. Did you manually confirm this changes fixes the issue.

Reply:
> What is the difference between previous line and this line.

The problem was mentioned in #33 

> Also, I see there are not tests for this repository. 

To do that, I've to test it against working database. the mocking might take long time for multiple components. 

> Did you manually confirm this changes fixes the issue.

Yes, I've tested this on the testnet including other edge cases.